### PR TITLE
 openstack-ardana: Add script to deploy ardana without jenkins (SCRD-4903)

### PR DESCRIPTION
**Pull request by flaviodsr**
This change adds a bash script that mimics the steps followed by the
openstack-ardana job to deploy ardana on ECP.

The intent of the script is to make easy deploying ardana on ECP from
the workstation (without the need of jenkins). This change also
adds a library that provides functions similar to the openstack-ardana
jenkins job steps, which can be called to execute the each step
individually.

### CODE REVIEW

File: scripts/jenkins/ardana/manual/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+ansible==2.7.7
+appdirs==1.4.3
+asn1crypto==0.24.0
+bcrypt==3.1.6
+certifi==2018.11.29
+cffi==1.12.1
+chardet==3.0.4
+cryptography==2.5
```
Comment (dirkmueller) [inline scripts/jenkins/ardana/manual/requirements.txt]: do we really need to have "frozen" dependencies here? I think we should have more liberal definitions.. why does it need to be locked to a specific version (same for all the other lines..)

Comment (flaviodsr) [reply scripts/jenkins/ardana/manual/requirements.txt]: I have no specific reason to have those frozen dependencies. Simple got it from the virtual environment.

Following your suggestion, I will make a change to improve that.

File: docs/ardana/testing.md [hunk 2]
```diff
@@ -181,17 +174,42 @@ To verify that the test environment is properly set up:
...
+### Manual Ardana deployment
+
+A bash script is provided to help deploying Ardana manually. The script sets up a virtual environment with ansible and all
+requirements and mimics the steps of the openstack-ardana job by calling the ansible playbooks in the appropriate
+sequence with the appropriate parameters, which are taken from `input.yml`.
+
+Before running the script you need to configure the parameters in the `input.yml` file to fit how you want ardana to be
+deployed. The options available are basically a subset of the inputs from the openstack-ardana jenkins job.
+
+**NOTE**: For building test packages from gerrit changes, the osc utility needs to be correctly installed and configured on
```
Comment (stefannica) [inline docs/ardana/testing.md]: Could you please move this note to the list under the `Test environment setup` section above ?

Comment (flaviodsr) [reply docs/ardana/testing.md]: done!

**Comment by stefannica** [comment]
I've been craving for something like this for a while now, but never had the time to do it. Great work.

**Pull request merged**